brooklinevoiceapp/lambda_function.py

```diff
@@ -50,8 +50,10 @@ def platform_to_mycity_request(event):
     mycity_request.request_id = event['request']['requestId']
     mycity_request.is_new_session = event['session']['new']
     mycity_request.session_id = event['session']['sessionId']
-    mycity_request.device_id = event['context']['System']['device']['deviceId']
-    mycity_request.api_access_token = event['context']['System']['apiAccessToken']
+
+    system_context = event['context']['System']
+    mycity_request.device_id = system_context.get('device', {}).get('deviceId', "unknown")

```

Comment:
Do you think we need to log when the deviceId or apiAccessToken are not valid?

Reply:
I don't think so. I think it would be the responsibility of any intent to make sure this value exists before trying to use it.